docker-compose/docker-compose.yaml

```diff
@@ -103,6 +103,10 @@ services:
       #
       # Uncomment / update the following line when using HTTPS with custom certificates
       # - '/LOCAL/KEY/PATH.key:/sourcegraph.key'
+      # 
+      # Uncomment / update the following line if Caddy is behind a reverse proxy or load balancer
+      # IMPORTANT: the file below needs to be updated to include customer's ips
+      # - '../caddy/builtins/https.trusted-proxies.Caddyfile'
```

Comment:
What happens if they're already using one of the other http or https configuration files?

Perhaps we just add this commented-out block in the existing files?

Reply:
I thought it bundled config files. My b, I shouldn't have assumed that. I'll clean this up later today.

Reply:
Fixed!